### PR TITLE
Rule Dev

### PR DESCRIPTION
**Pull request by nasbench**
This PR covers the following:

- Add more System Processes to the rule [d5866ddf-ce8f-4aea-b28e-d96485a20d3d](https://github.com/SigmaHQ/sigma/blob/15fc7f471133a504ffe2080ca63e8f15421e8eae/rules/windows/file/file_event/file_event_win_creation_system_file.yml)
- Also added `filter7` to the same rule because the 64bit version of `explorer.exe` is located in `C:\Windows\`
- Added more suspicious extension to the rule [d7b50671-d1ad-4871-aa60-5aa5b331fe04](https://github.com/SigmaHQ/sigma/blob/fac67328275e58413f299ed4f69219ff40803d70/rules/windows/file/file_event/file_event_win_new_files_in_uncommon_appdata_folder.yml)
- Renamed some `file_event` rules as they had a double `_win_` in their name.
  - [file_event_win_win_cscript_wscript_dropper.yml](https://github.com/SigmaHQ/sigma/blob/fac67328275e58413f299ed4f69219ff40803d70/rules/windows/file/file_event/file_event_win_win_cscript_wscript_dropper.yml)
  - [file_event_win_win_shell_write_susp_directory.yml](https://github.com/SigmaHQ/sigma/blob/fac67328275e58413f299ed4f69219ff40803d70/rules/windows/file/file_event/file_event_win_win_shell_write_susp_directory.yml)
  - [file_event_win_win_shell_write_susp_files_extensions.yml](https://github.com/SigmaHQ/sigma/blob/fac67328275e58413f299ed4f69219ff40803d70/rules/windows/file/file_event/file_event_win_win_shell_write_susp_files_extensions.yml)
- Updated the rule [79a87aa6-e4bd-42fc-a5bb-5e6fbdcd62f5](https://github.com/SigmaHQ/sigma/blob/1f8e37351e7c5d89ce7808391edaef34bd8db6c0/rules/windows/process_creation/proc_creation_win_msiexec_install_quiet.yml) to remove spaces from the arguments. As `msiexec` args can be used without spaces. I also increased the rule to medium since it focuses on silent installs.
- Added a new rule `8150732a-0c9d-4a99-82b9-9efb9b90c40c` related to `msiexec` that focus on installation from Remote Shares and Internet
- Added a new rule `a35c97c8-d9c4-4c89-a3e7-533dc0bcb73a` that focus on command line contents of files as arguments that have an extension with a `.` at the end (as seen used by raspberry robin) - Examples are linked in the rules
- Other metadata updates

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_raspberry_robin_single_dot_ending_file.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Raspberry Robin Dot Ending File
+id: a35c97c8-d9c4-4c89-a3e7-533dc0bcb73a
+status: experimental
+description: Detects commandline containing reference to files ending with a ".". This scheme has been seen used by raspberry-robin
+author: Nasreddine Bencherchali
+references:
+    - https://twitter.com/MichalKoczwara/status/1553634816016498688
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_raspberry_robin_single_dot_ending_file.yml:7]: https://www.microsoft.com/en-us/security/blog/2022/10/27/raspberry-robin-worm-part-of-larger-ecosystem-facilitating-pre-ransomware-activity/

File: rules/windows/process_creation/proc_creation_win_msiexec_install_remote.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            - '-package'
+            - '/a'
+            - '-a'
+            - '/j'
+            - '-j'
+    selection_quiet:
+        CommandLine|contains:
+            - '/q'
+            - '-q'
+    selection_remote:
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_msiexec_install_remote.yml:35]: keyword search or mlssing field name ?

File: rules/windows/process_creation/proc_creation_win_raspberry_robin_single_dot_ending_file.yml (hunk 1 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_raspberry_robin_single_dot_ending_file.yml:7]: When I pulled from master to resolve some conflicts, the ref got messed up. Thanks

File: rules/windows/process_creation/proc_creation_win_msiexec_install_remote.yml (hunk 2 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_msiexec_install_remote.yml:35]: Missing field name. Thanks for the catch

**Review by frack113 — APPROVED** [review]

**Pull request merged**